app/domain/authentication/authn_jwt/validate_status.rb

```diff
@@ -0,0 +1,123 @@
+module Authentication
+  module AuthnJwt
+
+    ValidateStatus = CommandClass.new(
+      dependencies: {
+        create_signing_key: Authentication::AuthnJwt::SigningKey::CreateSigningKeyInterface.new,
+        fetch_issuer_value: Authentication::AuthnJwt::ValidateAndDecode::FetchIssuerValue.new,
+        fetch_identity_from_token: Authentication::AuthnJwt::IdentityProviders::IdentityFromDecodedTokenProvider,
+        validate_webservice_is_whitelisted: ::Authentication::Security::ValidateWebserviceIsWhitelisted.new,
+        validate_role_can_access_webservice: ::Authentication::Security::ValidateRoleCanAccessWebservice.new,
+        validate_webservice_exists: ::Authentication::Security::ValidateWebserviceExists.new,
+        enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str,
```

Comment:
shouldn't this be an input as we do in the regular status check? what if an authenticator is enabled in the DB after the server has started to run?

Reply:
Fixed according to conversation with Sasha

Reply:
this should be an input